spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -267,7 +267,18 @@ class SpdxCompoundExpression(
  * An SPDX expression that contains only a single license with an optional exception. Can be
  * [SpdxLicenseWithExceptionExpression] or any subtype of [SpdxSimpleExpression].
  */
-sealed class SpdxSingleLicenseExpression : SpdxExpression()
+sealed class SpdxSingleLicenseExpression : SpdxExpression() {
+    companion object {
+        /**
+         * Parse a string into an [SpdxSingleLicenseExpression]. Throws an [SpdxException] if the string cannot be
+         * parsed.
+         */
+        @JsonCreator
+        @JvmStatic
+        fun parse(expression: String): SpdxSingleLicenseExpression =
+            parse(expression, Strictness.ALLOW_ANY) as SpdxSingleLicenseExpression
```

Comment:
You could drop the strictness here (like we do below for `SpdxLicenseWithExceptionExpression`) as that is the default.

Reply:
Just dropping it would result in an endless loop, because the function would be calling itself. But I replaced it with `SpdxExpression.parse()` instead of relying on the second parameter which is better because it is more explicit.